### PR TITLE
Update README.md to reflect #6 and #27

### PR DESCRIPTION
**Pull request by jsomedon**
Documenting workaround for no sound issue #6 and compilation issue caused by python version #27 for future users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ Play piano in the terminal using PC (computer) keyboard.
 
 ## Compiling
 
+Make sure `python` command is pointing to **python 2** but not python 3, [otherwise compilation may fail.](https://github.com/ritiek/piano-rs/issues/27)
```
Comment (ritiek) [inline README.md]: I tested this out linking my `python` to python 3.5 and python 3.6 but wasn't able to reproduce issue #27. I was able to reproduce this problem only with python 3.7. If so, it should be a good idea to mention about avoiding specifically python 3.7 and not python 3.x.

**Comment by ritiek** [comment]
I left a review about the problem in #27 being specific to python 3.7 but otherwise looks good to me!

**Comment by jsomedon** [comment]
Hmm, I don't see anything look like "review" in pull request page or issue page but, I guess you mean, instead of

> ... is pointing to **python 2** but not python 3 ...

you prefer

> ... is not pointing to python 3.7 ...

Is this what you mean?

---EDIT

And I just checked nsf/termbox#117 , so it looks like 3.7 is the specific version causing compilation issue. I'll rewrite that part.

**Comment by ritiek** [comment]
Yep, that's what I meant. Weird that it doesn't show up, I'll copy paste here:

"I tested this out linking my `python` to python 3.5 and python 3.6 but wasn't able to reproduce issue #27. I was able to reproduce this problem only with python 3.7. If so, it should be a good idea to mention about avoiding specifically python 3.7 and not python 3.x."

**Review by ritiek — APPROVED** [review]
Looks good to me. Thanks @jsomedon!

**Comment by jsomedon** [comment]
No problem! :smile:

**Pull request merged**